### DISCUSSION TITLE
Website stuck on loading for some users

### DISCUSSION THREAD

**Discussion by maysaleba** [community/community#27746] [Other Feature Feedback, Questions, & Ideas]
Hello,

Multiple users are reporting that our website (made with github pages) https://maysaleba.com/ are stuck on loading with no errors. The first few reports started yesterday, what's weird is that it is working fine for other people including myself. 

Anyone knows what can I check to identify this issue?

Thanks

**Comment by jfullstackdev** [comment]
you have several errors at the live website if you will look at the Developer Tools

![image](https://user-images.githubusercontent.com/47092464/182011516-a8d110b4-0a25-4cdb-b3f0-8319132410d7.png)

so maybe that's why it's affecting some users
